src/converter.ts

```diff
@@ -131,9 +131,54 @@ export default async function (ast: ParseResult<File>, targetedDiscordModApiLibr
          */
         debugger;
         // console.log(findAllTypesWithPath(element, "MemberExpression"));
-        const paths = findPathsToType({ obj: element, targetType: "MemberExpression" });
-        for (let index2 = 0; index2 < paths.length; index2++) {
-            const element2 = paths[index2];
+        const newExpressionPaths = findPathsToType({ obj: element, targetType: "NewExpression" });
+        for (let index2 = 0; index2 < newExpressionPaths.length; index2++) {
+            const element2 = newExpressionPaths[index2];
+            const trueObj = deepFind<NewExpression>(element, element2);
+            console.log(trueObj);
+            if (trueObj != undefined && importAliasMap.find(x => x.codeName == (trueObj.callee as Identifier).name) !== undefined) {
+                removeASTLocation(trueObj as unknown as Statement);
```

Comment:
Bad, don't

Reply:
what? how?

Reply:
`deepFind<Statement>` ?

Reply:
That's wrong, I need it to be `NewExpression` but it needs to be casted to `Statement` just for this call

Reply:
i guess removeAST... should take in input generic objects and not statements, idk to be fair

Reply:
Solved in c53d1947264c0f52d8dde4213e26377698f36c5f